backend/src/main/java/reviewme/member/domain/MemberStatus.java

```diff
@@ -0,0 +1,8 @@
+package reviewme.member.domain;
+
+public enum MemberStatus {
+
+    ACTIVE,
+    INACTIVE,
+    DELETED,
+}
```

Comment:
```suggestion
public enum MemberStatus {

    ACTIVE,
    INACTIVE,
    DELETED,
    ;
}
```

세미콜론이 없어도 컴파일 에러는 안나지만, 
다른 enum 들과 컨벤션을 통일해주는게 좋을 것 같아요~

Reply:
그리고 DELETED 와 INACTIVE 의 차이를 모르겠습니다..! 
설명해주세용~

Reply:
DELETED는 소프트 삭제(정책 상 탈되)인데, 보통 회원 같은 데이터는 민감해서 보통 바로 삭제하기 보다는 비활성 계정같은 삭제 전 단계로 두고, 특정 기한이 지나면 삭제되도록 해요. 당장 여기 상태값들은 사용될 일 없겠지만, 기본적으로 이렇게 존재하고, 나중에 추가로 필요한 상태값을 여기에 추가하는 식으로 하면 될 것 같습니다!

Reply:
아하
- INACTIVE : 삭제 전 단계
- DELETED : 삭제됨

이군요!